src/main/java/seedu/address/model/claim/Description.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.model.claim;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+public class Description {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Description of claim should only contain alphanumeric characters and spaces, and it should not be blank";
+
+    /*
+     * The first character of the description must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
+
+    public final String text;
+
+    /**
+     * Constructs a {@code Description}.
+     *
+     * @param text
```

Comment:
javadoc comment should explain more about what the method is supposed to achieve